Source/AwsCommonRuntimeKit/crt/CRTErrorGenerated.swift

```diff
@@ -0,0 +1,307 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0.
+// This file is generated using Script/CRTErrorGenerator.swift.
+// Do not modify this file.
+
+import AwsCCommon
+
+/// Error type for CRT errors thrown from C code
+public enum CRTError: Int32, Error {
```

Comment:
trivial: similar, is it worth making it `Int` instead of `Int32` just to be more swift-like?